packages/dart/npt_flutter/lib/localization/app_en.arb

```diff
@@ -61,11 +61,10 @@
    "submit" : "Submit",
    "validationErrorEmptyField" : "Field cannot be left Blank",
    "validationErrorAtsignField" : "Field must be a valid atsign that starts with @",
-   "validationErrorProfileNameField" : "Field can only contain lowercase letters, digits, and spaces.",
    "validationErrorDeviceNameField" : "Field can only contain lowercase letters, digits, underscores.",
    "validationErrorLongField" : "Field must be 1-36 characters long",
    "validationErrorLocalPortField" : "Field must be 0 or between 1024-65535",
-   "validationErrorRemotePortField" : "Field must be between 1024-65535",
```

Comment:
Local port cannot be 0 in the desktop app, maybe in the future but not supported today